kuma/wiki/tests/test_feeds.py

```diff
@@ -49,10 +51,10 @@ def test_updated_translation_parent_feed_mod_link(self):
         eq_(1, len(feed.find('item')))
         for i, item in enumerate(feed.find('item')):
             desc_text = pq(item).find('description').text()
-            ok_("%s$compare?to=%s&from=%s" % (d1.slug,
-                                              d1.current_revision.id,
-                                              first_rev_id)
-                in desc_text)
+            compare_url = '%s$compare?to=%s&from=%s' % (d1.slug,
+                                                        d1.current_revision.id,
+                                                        first_rev_id)
+            ok_(escape(compare_url) in desc_text)
```

Comment:
Oh yeah, forgot we decided to do that. I'll update to use `assertTrue` instead of `ok_` and `assertEqual` instead of `eq_`.